examples/rax-kitchen-sink/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "rax-kitchen-sink",
+  "author": "Thanh",
+  "version": "5.0.0-beta.3",
+  "scripts": {
+    "start": "rax-scripts start",
+    "build": "rax-scripts build",
+    "lint": "eslint --ext .js --ext .jsx ./src",
+    "storybook": "start-storybook -p 9009 -s public",
+    "build-storybook": "build-storybook -s public"
+  },
+  "dependencies": {
+    "rax": "^0.6.5",
+    "rax-button": "^0.6.5",
+    "rax-image": "^0.6.8",
+    "rax-link": "^0.6.5",
+    "rax-text": "^0.6.5",
+    "rax-view": "^0.6.5"
+  },
+  "devDependencies": {
+    "@storybook/addon-a11y": "5.0.0-beta.3",
+    "@storybook/addon-actions": "5.0.0-beta.3",
+    "@storybook/addon-backgrounds": "5.0.0-beta.3",
+    "@storybook/addon-centered": "5.0.0-beta.3",
+    "@storybook/addon-events": "5.0.0-beta.3",
+    "@storybook/addon-info": "5.0.0-beta.3",
+    "@storybook/addon-jest": "5.0.0-beta.3",
+    "@storybook/addon-knobs": "5.0.0-beta.3",
+    "@storybook/addon-links": "5.0.0-beta.3",
+    "@storybook/addon-notes": "5.0.0-beta.3",
+    "@storybook/addon-options": "5.0.0-beta.3",
+    "@storybook/addon-storyshots": "5.0.0-beta.3",
```

Comment:
Why no `@storybook/rax` dep?

Reply:
Great catch. I've copied from other examples so lots of dependencies to compare and remove/add. I've got the rax-kitchen-sink working with basic button example for now. 

Thanks.